### PR TITLE
Documentation: Consoleify cat shards/recovery API docs

### PR DESCRIPTION
**Pull request by spinscale**
Relates #23001

### CODE REVIEW

File: docs/reference/cat/recovery.asciidoc [hunk 1]
```diff
@@ -12,17 +12,20 @@ way for shards to be loaded from disk when a node starts up.
...
-index 4     9ms  store done  127.0.0.1        I8hydUG      127.0.0.1        I8hydUG      n/a        n/a      0     0.0%          0     0.0%          0           0           0        100.0%           0
+GET _cat/recovery?v
+----------------------------------------------------------------------------
+// CONSOLE
+// TEST[setup:big_twitter]
+
+[source,js]
+----------------------------------------------------------------------------
+index shard time type stage source_host source_node target_host target_node repository snapshot files files_recovered files_percent files_total bytes bytes_recovered bytes_percent bytes_total translog_ops translog_ops_recovered translog_ops_percent
+twitter 0     18ms empty_store done  n/a         n/a         127.0.0.1   node-0      n/a        n/a      0     0               0.0%          0           0     0               0.0%          0           0            0                      100.0%
```
Comment (nik9000) [inline docs/reference/cat/recovery.asciidoc]: `empty_store`? Do we have a caching issue here? If so then this is going to be unstable....

File: docs/reference/cat/recovery.asciidoc [hunk 2]
```diff
@@ -31,20 +34,30 @@ Now let's see what a live recovery looks like. By increasing the replica count
...
-> curl -XGET 'localhost:9200/_cat/recovery?v&h=i,s,t,ty,st,shost,thost,f,fp,b,bp'
-i     s t      ty      st    shost  thost  f     fp      b        bp
-wiki  0 1252ms store   done  hostA  hostA  4     100.0%  23638870 100.0%
-wiki  0 1672ms replica index hostA  hostB  4     75.0%   23638870 48.8%
-wiki  1 1698ms replica index hostA  hostB  4     75.0%   23348540 49.4%
-wiki  1 4812ms store   done  hostA  hostA  33    100.0%  24501912 100.0%
-wiki  2 1689ms replica index hostA  hostB  4     75.0%   28681851 40.2%
-wiki  2 5317ms store   done  hostA  hostA  36    100.0%  30267222 100.0%
+i       s t      ty             st   shost thost f fp     b        bp
+twitter 0 1252ms existing_store done n/a   hostA 4 100.0% 23638870 0.0%
```
Comment (nik9000) [inline docs/reference/cat/recovery.asciidoc]: I'd prefer this tell a story in line with the first one but it doesn't really. Either we should use fake data and replace the whole time or we should use real data.

File: docs/reference/cat/recovery.asciidoc (hunk 1 above)
Comment (spinscale) [inline docs/reference/cat/recovery.asciidoc]: I dont know TBH. This setting remains `empty_store` when you create an index in a running cluster, but when I restart the cluster that holds this index, then the value becomes `existing_store`. I thought this was intended

File: docs/reference/cat/recovery.asciidoc [hunk 3]
```diff
@@ -12,17 +12,22 @@ way for shards to be loaded from disk when a node starts up.
 As an example, here is what the recovery state of a cluster may look like when there
 are no shards in transit from one node to another:
 
-[source,sh]
+[source,text]
```
Comment (nik9000) [inline docs/reference/cat/recovery.asciidoc]: We use `source,js` for these. We might should use `source,console` but we use `source,js`. These are just markers for us. The syntax highlighter infers the language at page load.

File: docs/reference/cat/recovery.asciidoc [hunk 4]
```diff
@@ -54,20 +60,16 @@ Finally, let's see what a snapshot recovery looks like. Assuming I have previous
...
-imdb  1 2790ms snapshot index imdb       snap_1   88    7.7%    11025 8.1%
-imdb  2 2790ms snapshot index imdb       snap_1   85    0.0%    12072 0.0%
-imdb  3 2796ms snapshot index imdb       snap_1   85    2.4%    12048 7.2%
-imdb  4  819ms snapshot init  imdb       snap_1   0     0.0%    0     0.0%
+[source,text]
 --------------------------------------------------------------------------------
+GET _cat/recovery?v&h=i,s,t,ty,st,rep,snap,f,fp,b,bp
+---------------------------------------------------------------------------
+// CONSOLE
+// TEST[skip:no need to execute snapshot/restore here]
```
Comment (nik9000) [inline docs/reference/cat/recovery.asciidoc]: I think we should add text like "Which returns:" or something between these two snippets.

Comment (nik9000) [inline docs/reference/cat/recovery.asciidoc]: And the other similar snippets

File: docs/reference/cat/shards.asciidoc [hunk 5]
```diff
@@ -7,13 +7,22 @@ docs, the bytes it takes on disk, and the node where it's located.
...
+---------------------------------------------------------------------------
+// CONSOLE
+// TEST[setup:twitter]
+
+[source,text]
+---------------------------------------------------------------------------
+twitter 0 p STARTED 3014 31.1mb 192.168.56.10 H5dfFeA
+---------------------------------------------------------------------------
+// TESTRESPONSE[s/3014/\\d+/]
+// TESTRESPONSE[s/31.1/\\d+\.\\d+/]
```
Comment (nik9000) [inline docs/reference/cat/shards.asciidoc]: I'd combine the `mb` one with this one.

**Review by nik9000 — APPROVED** [review]
I left a few minor things but I think it is nicer. Thanks so much!

**Comment by spinscale** [comment]
added correct source types, added text between snippets and also fixed some test that was left over

**Review by nik9000 — APPROVED** [review]

**Pull request merged**